mkdocs.yml

```diff
@@ -0,0 +1,69 @@
+site_name: C/C++ Linter Action's Docs
+site_description: "Developer documentation from sources."
+site_url: "https://shenxianpeng.github.io/cpp-linter-action"
+repo_url: "https://github.com/shenxianpeng/cpp-linter-action"
+edit_uri: "blob/master/docs/"
+repo_name: "shenxianpeng/cpp-linter-action"
+nav:
+  - index.md
+  - "Dev Docs":
+    - API Reference/python_action.md
```

Comment:
I tend to use an indent of 2 spaces in yml files. Not sure what code-inspector is assuming, but its suggestion is wrong (again).

Reply:
I have code inspector set to ignore this file. If we switch to using Sphinx instead of mkdocs, then this won't be needed. 

Note that docs are built using RTFD.org's CI anyway